install/kubernetes/cilium/values.yaml

```diff
@@ -82,6 +82,12 @@ global:
     serviceMonitor:
       enabled: false
 
+  # enableXTSocketFallback enables the fallback compatibility solution when
+  # the xt_socket kernel module is missing and it is needed for the datapath
+  # L7 redirection to work properly.  See documentation for details on when
+  # this can be disabled.
```

Comment:
I think it would be useful to have a specific link here to provide the background.

This is probably accurate enough assuming that we settle on the current documentation approach:
http://docs.cilium.io/en/latest/install/system_requirements/#admin-kernel-version

Reply:
Link added :-)